kubernetes/structure_service_spec.go

```diff
@@ -54,6 +54,10 @@ func flattenServiceSpec(in v1.ServiceSpec) []interface{} {
 	if in.ExternalName != "" {
 		att["external_name"] = in.ExternalName
 	}
+	if in.PublishNotReadyAddresses != false {
```

Comment:
Why do we need this check?

Reply:
I was asking myself the same question then I took example from https://github.com/terraform-providers/terraform-provider-kubernetes/blob/master/kubernetes/structure_persistent_volume_spec.go#L20

Reply:
Removed the check.